blockchain/ethereum/chain.go

```diff
@@ -52,6 +53,13 @@ func NewChainBackend(url string, chainConnTimeout, onChainTxTimeout time.Duratio
 	if err = ks.Unlock(acc, cred.Password); err != nil {
 		return nil, errors.Wrap(err, "unlocking on-chain keystore for addr - "+cred.Addr.String())
 	}
-	cb := pethchannel.NewContractBackend(ethereumBackend, ks, &acc)
+
+	// Create keystore wallet
```

Comment:
```suggestion
```

Reply:
Deleted in this [commit](https://github.com/hyperledger-labs/perun-node/pull/110/commits/419c09f1a62eb87b03be5594f43da29dfd7ed31c)